### PR TITLE
Remove streamlined waiver asset update

### PR DESCRIPTION
**Pull request by amponce**
## Summary

Financial_status_report_streamlined_waiver_assets has been removed for some time now, it's safe to remove any remaining references to it via `streamlinedWaiverAssetUpdate`. The flag has been defaulted to true

Flag is defaulted to true in src/applications/financial-status-report/containers/App.jsx:155

https://github.com/department-of-veterans-affairs/vets-website/pull/28078
Feature flag clean up tickets:
[FSR] Deprecate unused enhanced feature flag [#94930](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/94930)
[FSR] Deprecate unused streamlinedwaiver feature flag [#94931](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/94931)
[FSR] Deprecate unused streamlinedWaiverAssetUpdate feature flag [#94932](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/94932) (this one)

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#94932

## Testing done

- Manual Testing
- Verified unit and cypress tests pass as expected

### CODE REVIEW

**Review by rmessina1010 — DISMISSED** [review]
LGTM

File: src/applications/financial-status-report/mocks/development.js
Comment (Dr-Pongo) [inline src/applications/financial-status-report/mocks/development.js:1]: I think we want to keep this defaulted to false!

**Review by Dr-Pongo — CHANGES_REQUESTED** [review]
I think we should keep that development flag defaulted to false, but otherwise looks good!

**Review by josephrlee — DISMISSED** [review]

**Review by Dr-Pongo — APPROVED** [review]
LGTM! 🚀

**Pull request merged**